tests/acceptance/features/apiSpacesDavOperation/copyByFileId.feature

```diff
@@ -0,0 +1,66 @@
+Feature: copying file using file id
+  As a user
+  I want to copy the file using file id
+  So that I can manage my resource
+
+  Background:
+    Given using spaces DAV path
+    And user "Alice" has been created with default attributes and without skeleton files
+
+
+  Scenario Outline: copy a file from folder to the root of personal space
```

Comment:
Should this be like this because secanrio 40 also has same scenario title
```suggestion
  Scenario Outline: copy a file into a folder of personal space
```
```suggestion
  Scenario Outline: copy a file from root to the folder of personal space
```